tests/lib/fixtures.py

```diff
@@ -73,23 +74,25 @@ def authentication_cache():
 
 
 @pytest.fixture
-def bearer_token_internal_dev(authentication_cache, api_product_in_comms_manager_local):
-    return authentication_cache.generate_authentication('internal-dev')
+def bearer_token_internal_dev(authentication_cache, api_product_in_comms_manager_local, nhsd_apim_proxy_url):
+    return authentication_cache.generate_authentication('internal-dev', nhsd_apim_proxy_url)
 
 
 @pytest.fixture
-def bearer_token_internal_dev_test_1(authentication_cache, api_product_in_nhs_notify_test_client_1):
-    return authentication_cache.generate_authentication('internal-dev-test-1')
+def bearer_token_internal_dev_test_1(authentication_cache,
+                                     api_product_in_nhs_notify_test_client_1,
+                                     nhsd_apim_proxy_url):
```

Comment:
This call signature has been updated but it doesn't look like you've updated anywhere that it's used. Can this method be removed or is there some smart parameter passing going on when this method gets used?

Reply:
This fixture is used by rate limiting tests: https://github.com/NHSDigital/communications-manager-api/blob/fix/CCM-6648_Intermittent_401_errors/tests/development/rate_limiting/test_429_errors.py

The fixture is injected as a test parameter and pytest will build it using this function.

Happy to explain on a call!

Reply:
Ah neat, I thought it might be something clever under the hood